### DISCUSSION TITLE
如何使用指定版本的vs工具链

### DISCUSSION THREAD

**Discussion by liaozixin** [xmake-io/xmake#2319] [Q&A] [answered]
使用--bin选项定位到vs2019的VC bin目录下，但构建时还是vs2022。请问需要如何指定
`xmake f --toolchain=msvc --bin="D:\VS2019\VC\Tools\MSVC\14.29.30133\bin\Hostx64\x64"`

**Comment by waruqi** [comment, marked as answer]
xmake f --vs=2019 -c

**Reply by liaozixin** [reply]
感谢您的帮助
